packager/mpd/base/xml/xml_node_unittest.cc

```diff
@@ -368,13 +368,14 @@ TEST_F(LiveSegmentTimelineTest, OneSegmentInfo) {
   const uint64_t kStartTime = 0;
   const uint64_t kDuration = 100;
   const uint64_t kRepeat = 9;
+  const bool kIsLowLatency = false;
```

Comment:
Looks like you have disabled low latency mode for UT. Is it not possible to add some test cases for low latency mode?

Reply:
The unit tests are coming up soon! :)